add new sso and scim faqs (#8963)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/scim.md

- You must be on an [Enterprise or Enterprise+ plan](https://www.getdbt.com/pricing).
- You must use Okta or Entra ID as your SSO provider and have it connected in the dbt platform.
- You must have [permissions](/docs/cloud/manage-access/enterprise-permissions) to configure the account settings in <Constant name="dbt_platform" /> and change application settings in [Okta](https://help.okta.com/en-us/content/topics/security/administrators-admin-comparison.htm).

> - If you have IP restrictions enabled, you must add [Okta's IPs](https://help.okta.com/en-us/content/topics/security/ip-address-allow-listing.htm) to your allowlist. If you're using Entra ID with IP restrictions enabled, see [Azure SCIM provisioning fails due to IP allowlisting](/docs/cloud/manage-access/scim-faq) in the SCIM FAQ.

Reviewer: ```suggestion
- If you have IP restrictions enabled, you must add [Okta's IPs](https://help.okta.com/en-us/content/topics/security/ip-address-allow-listing.htm) to your allowlist. If you're using Entra ID with IP restrictions enabled, refer to [Azure SCIM provisioning fails due to IP allowlisting](/docs/cloud/manage-access/scim-faq) in the SCIM FAQ.
```

When SCIM is enabled, the following functionality will change:
- Users are not automatically added to default groups
- Manual actions such as inviting users, updating user information and updating group memberships are disabled by default

> - SSO group mappings are disabled in favor of SCIM group management. Refer to [Do SSO group mappings still apply when SCIM is enabled?](/docs/cloud/manage-access/scim-faq) for details

Reviewer: ```suggestion
- SSO group mappings are disabled in favor of SCIM group management. Refer to [Do SSO group mappings still apply when SCIM is enabled?](/docs/cloud/manage-access/scim-faq) for details.
```

7. Copy the token and record it securely, as _it will not be available again after you close the window_. You must create a new token if you lose the current one.
    <Lightbox src="/img/docs/dbt-cloud/access-control/copy-scim-token.png" width="60%" title="Give your token and identifier." />
8. (Optional) Manual updates are turned off by default for all SCIM-managed entities, including the ability to invite new users manually. This ensures SCIM-managed entities stay in sync with the IdP, and we recommend keeping this setting disabled.

>    - However, if you need to make manual updates (like update group membership for a SCIM-managed group), you can enable this setting by clicking **Allow manual updates** and confirming the **Allow manual updates** pop up. For more details on this setting, see [What does "Allow manual updates" mean?](/docs/cloud/manage-access/scim-faq) in the SCIM FAQ.

Reviewer: ```suggestion
   - However, if you need to make manual updates (like update group membership for a SCIM-managed group), you can enable this setting by clicking **Allow manual updates** and confirming the **Allow manual updates** pop up. For more details on this setting, refer to [What does "Allow manual updates" mean?](/docs/cloud/manage-access/scim-faq?version=1.12#what-does-allow-manual-updates-mean) in the SCIM FAQ.
```

Reviewer: Is it possible to link to the relevant toggle?

Author: good catch!

- **[Set up SCIM with Okta](/docs/cloud/manage-access/scim-okta)** &mdash; User and group provisioning, profile updates, and [license management](/docs/cloud/manage-access/scim-manage-user-licenses) (Okta only).
- **[Set up SCIM with Entra ID](/docs/cloud/manage-access/scim-entra-id)** &mdash; User and group provisioning and profile updates, plus assigning users to the SCIM app.

> - **[SCIM FAQ and troubleshooting](/docs/cloud/manage-access/scim-faq)** &mdash; Common questions and troubleshooting for SCIM provisioning.

Author: ```suggestion
- **[SCIM FAQs and troubleshooting](/docs/cloud/manage-access/scim-faq)** &mdash; Common questions and troubleshooting for SCIM provisioning.
```
